src/phase0/state_transition/epoch_processing.rs

```diff
@@ -254,8 +254,25 @@ pub fn process_effective_balance_updates<
         MAX_VALIDATORS_PER_COMMITTEE,
         PENDING_ATTESTATIONS_BOUND,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<(), Error> {
+    // Update effective balances with hysteresis
+    for i in 0..state.validators.len() {
+        let validator = &mut state.validators[i];
+        let balance = &state.balances[i];
```

Comment:
balance is a `Gwei` which is `Copy` so you can just do an assign here:
```suggestion
        let balance = state.balances[i];
```

and that will simplify everything else :)

Reply:
Ahh that's right, ugh. Got it.